src/types/investment.ts

```diff
@@ -119,27 +119,27 @@ export type InvestmentRateType = typeof INVESTMENT_RATE_TYPES[number]
 
 export type Expenses = {
   /** Service tax that varies according to state */
-  serviceTax?: number
+  serviceTax?: number | null
   /** Commission charged by the brokerage for carrying out transactions on the stock market */
-  brokerageFee?: number
+  brokerageFee?: number | null
   /** Income Tax Withholding, amount paid to the Internal Revenue Service */
-  incomeTax?: number
+  incomeTax?: number | null
   /** Sum of other not defined expenses */
-  other?: number
+  other?: number | null
   /** Fee of Notice of Trading in Assets */
-  tradingAssetsNoticeFee?: number
+  tradingAssetsNoticeFee?: number | null
   /** Fees charged by BM&F Bovespa in negotiations */
-  maintenanceFee?: number
+  maintenanceFee?: number | null
   /** Liquidation fee for the settlement of a position on the expiration date or the financial settlement of physical delivery */
-  settlementFee?: number
+  settlementFee?: number | null
   /** Registration fee */
-  clearingFee?: number
+  clearingFee?: number | null
   /** Fees charged by BM&F Bovespa as a source of operating income  */
-  stockExchangeFee?: number
+  stockExchangeFee?: number | null
   /** Fee by brokers to keep recordsin their home broker systems or on the trading desk */
-  custodyFee?: number
+  custodyFee?: number | null
   /** Amount paid to the Operator for the intermediation service  */
-  operatingFee?: number
+  operatingFee?: number | null
```

Comment:
are all these fields correct as both optional _and_ nullable? (`| null`)

Reply:
No, just the `Identity` object is _optional_ and  _nullable_, my bad 👍

Reply:
This matches with the API response

Reply:
Fixed! Now it's just _nullable_